proxy/handler.go

```diff
@@ -0,0 +1,52 @@
+package proxy
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/ONSdigital/dp-legacy-cache-proxy/config"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+func (proxy *Proxy) Handler(ctx context.Context, w http.ResponseWriter, req *http.Request, cfg *config.Config) {
+	targetURL := cfg.BabbageURL + req.URL.String()
+
+	client := &http.Client{}
+
+	proxyReq, err := http.NewRequestWithContext(ctx, req.Method, targetURL, req.Body)
+	if err != nil {
+		log.Error(ctx, "proxyHandler: Error constructing proxy request", err)
```

Comment:
Just a reminder that all these error logs would need refactoring when you change this method's name.